vlib/x/json2/json_module_compatibility_test/json_todo_test.vv

```diff
@@ -330,40 +316,3 @@ fn test_decode_missing_maps_field() {
 	assert '${info.items}' == '[]'
 	assert '${info.maps}' == '{}'
 }
-
-struct Foo3 {
-	name string
-	age  int    @[omitempty]
-}
-
-//! BUGFIX - .from_json(res)
-fn test_omit_empty() {
-	foo := Foo3{'Bob', 0}
-	assert json.encode_pretty(foo) == '{
-  "name": "Bob"
-}'
-}
-
-struct Asdasd {
-	data GamePacketData
-}
-
-type GamePacketData = GPEquipItem | GPScale
-
-struct GPScale {
-	value f32
-}
-
-struct GPEquipItem {
-	name string
-}
-
-fn create_game_packet(data &GamePacketData) string {
-	return json.encode(data)
-}
-
-//! FIX: returning null
-fn test_encode_sumtype_defined_ahead() {
-	ret := create_game_packet(&GamePacketData(GPScale{}))
-	assert ret == '{"value":0,"_type":"GPScale"}'
-}
```

Comment:
Why delete existing tests at all, in a PR named "cleanup"???

Reply:
It is a .vv file. I remove from there to be put at .v file